README.md

```diff
@@ -379,4 +379,15 @@ git stash pop
 ```
 
 ---
+## SSH Keys
+Note: This section is if you are using **SSH keys to clone your repo** and have already generated your GitHub SSH keys and ran your keygen in your terminal (console of your computer)
 
+ 
+A common problem you face when setting the SSH key on mac is when you restart your computer the ssh key makes you authenticate with your passphrase. 
```

Comment:
Small writing TIp.

..................... on the Mac is that when you restart your computer .....

Reply:
Small Tip: when nitpicking please add "Nitpick:"